introduction.md

```diff
@@ -1,3 +1,14 @@
 Hello, this is Ivan Reshetnikov. I greet you sensei with great honour.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?